source/jquery.flot.hover.js

```diff
@@ -152,12 +152,22 @@ the tooltip from webcharts).
             pos.pageX = page.X;
             pos.pageY = page.Y;
 
-            var item = plot.findNearbyItem(canvasX, canvasY, seriesFilter, distance);
+            var items = plot.findNearbyItems(canvasX, canvasY, seriesFilter, distance);
+            var item = items[0];
+
+            for (var i = 1; i < items.length; ++i) {
+            if (item.distance === undefined || 
```

Comment:
Not sure if this spacing is actually wrong or it's just github displaying it wrong.

Reply:
Fixed.